src/templates/static/metatrader/download.jsx

```diff
@@ -198,6 +198,7 @@ const MT5WebPlatform = ({ is_first_child }) => (
                 <span>{it.L('Trade with Real account')}</span>
             </a>
             <a
+                id='btn_trade_with_demo'
```

Comment:
is this intentional?

Reply:
@mustofa-binary https://github.com/binary-com/binary-static/pull/5971/files#diff-a34c6a1ccc85212a993f3010dafeb4f3R158